.gitignore

```diff
@@ -1,3 +1,7 @@
 .DS_Store
 node_modules/
 out/
+
+# Editor files
+.vscode
```

Comment:
You usally want to commit the `.vscode` folder. Here we need `launch.json` checked in.
```suggestion
```